ydb/core/blobstorage/dsproxy/root_cause.h

```diff
@@ -59,8 +61,22 @@ struct TRootCause {
 #endif //LWTRACE_DISABLE
     }
 
+    TString ToString() {
+        TStringStream str("TRootCause {");
+        str << " CurrentCauseIdx# " << CurrentCauseIdx;
+        str << " Items# [";
+        for (const TRootCauseItem& item : Items) {
+            str << " { Timestamp# " << item.Timestamp
+                << " IsAccelerate# " << item.IsAccelerate
+                << " CauseIdx# " << item.CauseIdx
```

Comment:
What does CauseIdx mean? I can't figure it out even by looking at the code. Maybe consider using human-readable strings for reasons of acceleration?